src/main/java/hw26/TestRunner.java

```diff
@@ -0,0 +1,72 @@
+package hw26;
+
+import hw26.annotations.AfterSuite;
+import hw26.annotations.BeforeSuite;
+import hw26.annotations.Priority;
+import hw26.annotations.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+
+public class TestRunner {
+    public static void start(Class<?> clazz) {
+        Objects.requireNonNull(clazz);
+        try {
+            checkSuiteAnnotationDuplicates(clazz);
+            AnnotatedMethodContainer annotatedMethod = getMethod(clazz);
+            Object instance = clazz.getDeclaredConstructor().newInstance();
+            annotatedMethod.beforeSuiteMethods().forEach(method -> invokeMethod(method, instance));
```

Comment:
Would be better to check if the `beforeSuiteMethods` size is not empty and if not then invoke the methods. Do the same for `testMethods` and `afterSuiteMethods`